snap/snapcraft.yaml

```diff
@@ -58,16 +61,23 @@ apps:
 
 parts:
   spark:
-    plugin: nil
+    plugin: python
+    python-packages:
+      # - https://github.com/canonical/spark-k8s-toolkit-py/releases/download/v0.0.1/spark8t-${SPARK8T_VERSION}-py3-none-any.whl
```

Comment:
Please remove the commented line :)

Reply:
**suggestion** I don't mind too much. I mean, once we are confident that the refactoring is stable, we should probably revert on using the tar.gz instead of the branch. Up to @juditnovak (being a suggestion)

Reply:
So THIS is why we would need that variable.

Frankly, I don't mind at all what we wanna do on this. I just thought that soon enough we'll switch to a versioned, packaged reference to the `spark8t` package instead of the current, pretty fragile solution taking the latest `main/HEAD`.

The line above, together with the variable pointing to the desired `spark8t` version is to facilitate this future step.

I can either add a comment above, like 
```
# Location of the spark8t package for future reference
```
or I can completely remove it, I really don't mind/care. This was just to facilitate a soon upcoming change.

Reply:
Let's keep it such that the comments will hurt our OCD around these things, and we will switched to the versioned one sooner rather than later

Reply:
See https://github.com/canonical/spark-client-snap/pull/62#discussion_r1205277780 removed for now.